deltachat-rpc-client/src/deltachat_rpc_client/contact.py

```diff
@@ -0,0 +1,32 @@
+class Contact:
+    """
+    Contact API.
+
+    Essentially a wrapper for RPC, account ID and a contact ID.
+    """
+    def __init__(self, rpc, account_id, contact_id):
+        self.rpc = rpc
+        self.account_id = account_id
+        self.contact_id = contact_id
+
+    async def block(self):
+        """Block contact."""
+        await self.rpc.block_contact(self.account_id, self.contact_id)
+
+    async def unblock(self):
+        """Unblock contact."""
+        await self.rpc.unblock_contact(self.account_id, self.contact_id)
+
+    async def delete(self):
+        """Delete contact."""
+        await self.rpc.delete_contact(self.account_id, self.contact_id)
+
+    async def change_name(self, name: str):
+        await self.rpc.change_contact_name(self.account_id, self.contact_id, name)
```

Comment:
users might expect that this mutates the existing contact class instance, what can we do against that expectation?